File: examples/ruby/constructs/loop.rb
# Ported from examples/lumen/constructs/loop.lm by scripts/port_examples.py; edit the Lumen original, not this file.
x = 0
while x < 10 do
    print(x)
    if x < 9 then
        print(", ")
    end
    x = x + 1
end
puts("")

File: examples/rust/constructs/loop.rs
// Ported from examples/lumen/constructs/loop.lm by scripts/port_examples.py; edit the Lumen original, not this file.
fn main() {
    let mut x = 0;
    while x < 10 {
        print!("{}", x);
        if x < 9 {
            print!(", ");
        }
        x = x + 1;
    }
    println!("");
}
